# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../wvstreams/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur wvstreams-4.2.2-orig/include/uniconfkey.h wvstreams-4.2.2/include/uniconfkey.h
--- wvstreams-4.2.2-orig/include/uniconfkey.h	2005-11-24 19:46:46.000000000 +0000
+++ wvstreams-4.2.2/include/uniconfkey.h	2006-04-02 06:22:18.356916500 +0000
@@ -113,7 +113,7 @@
     bool iswild() const;
 
     /** Returns true if the key has a trailing slash. */
-    bool UniConfKey::hastrailingslash() const;
+    bool hastrailingslash() const;
 
     /**
      * Returns the number of segments in this path.
diff -Nur wvstreams-4.2.2-orig/include/wvhttppool.h wvstreams-4.2.2/include/wvhttppool.h
--- wvstreams-4.2.2-orig/include/wvhttppool.h	2005-11-24 19:46:46.000000000 +0000
+++ wvstreams-4.2.2/include/wvhttppool.h	2006-04-02 06:29:53.689373000 +0000
@@ -57,7 +57,7 @@
     WvString linkname;
     WvUrl url;
 
-    WvUrlLink::WvUrlLink(WvStringParm _linkname, WvStringParm _url)
+    WvUrlLink(WvStringParm _linkname, WvStringParm _url)
 	: linkname(_linkname), url(_url)
     {}
 };
diff -Nur wvstreams-4.2.2-orig/include/wvstring.h wvstreams-4.2.2/include/wvstring.h
--- wvstreams-4.2.2-orig/include/wvstring.h	2005-11-24 19:46:46.000000000 +0000
+++ wvstreams-4.2.2/include/wvstring.h	2006-04-02 06:19:47.519489750 +0000
@@ -393,7 +393,7 @@
     
 protected:
     void copy_constructor(const WvFastString &s);
-    inline void WvString::construct(const char *_str)
+    inline void construct(const char *_str)
         {
             link(&nullbuf, _str);
     
diff -Nur wvstreams-4.2.2-orig/include/wvx509.h wvstreams-4.2.2/include/wvx509.h
--- wvstreams-4.2.2-orig/include/wvx509.h	2006-01-09 21:42:53.000000000 +0000
+++ wvstreams-4.2.2/include/wvx509.h	2006-04-02 06:25:47.621994750 +0000
@@ -349,19 +349,19 @@
      * Set the Certificate to use X509v3, since that's all modern
      * PKI uses anyways :)
      */
-    void WvX509Mgr::set_version();
+    void set_version();
 
     /**
      * Get and set the keyUsage field.
      */
-    WvString WvX509Mgr::get_key_usage();
-    void WvX509Mgr::set_key_usage(WvStringParm values);
+    WvString get_key_usage();
+    void set_key_usage(WvStringParm values);
 
     /**
      * Get and set the extendedKeyUsage field.
      */
-    WvString WvX509Mgr::get_ext_key_usage();
-    void WvX509Mgr::set_ext_key_usage(WvStringParm values);
+    WvString get_ext_key_usage();
+    void set_ext_key_usage(WvStringParm values);
     
     /**
      * Return the Subject alt name if it exists, and WvString::null if
